roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -148,17 +142,17 @@ def is_available_skopeo_image(self, image, registry, task_vars):
             "cleanup": True,
         }
         result = self.module_executor("docker_container", args, task_vars)
-        return result.get("failed", False)
+        return not result.get("failed", False)
```

Comment:
The logic was wrong. The method's name is `is_available_skopeo_image`. We expect a return value of `true` if the module_executor's result did *not* fail